core/src/main/java/com/novoda/landingstrip/ScrollOffsetCalculator.java

```diff
@@ -1,26 +1,29 @@
 package com.novoda.landingstrip;
 
 import android.view.View;
+import android.view.ViewGroup;
 
 class ScrollOffsetCalculator {
 
     private static final float HALF_MULTIPLIER = 0.5f;
     private static final float ROUNDING_OFFSET = 0.5f;
 
-    private final TabsContainer tabsContainer;
+    private final View rootView;
+    private final ViewGroup tabsContainer;
 
-    ScrollOffsetCalculator(TabsContainer tabsContainer) {
+    ScrollOffsetCalculator(View rootView, TabsContainerView tabsContainer) {
+        this.rootView = rootView;
         this.tabsContainer = tabsContainer;
     }
 
     int calculateScrollOffset(int position, float pagerOffset) {
-        if (!tabsContainer.hasTabAt(position)) {
+        if (!hasTabAt(position)) {
```

Comment:
nah @joetimmins fixed this with `return position < tabsContainer.getChildCount();`